tests/SimplifyTest.elm

```diff
@@ -21526,7 +21526,9 @@ setIntersectTests =
             \() ->
                 """module A exposing (..)
 import Set
-a = Set.intersect set set
```

Comment:
`Set.insersect set set` should actually be reported, as it can be simplified to `set` (same for `Set.union`, and I imagine all the same things for `Dict`). I'll add them to #2.

Reply:
Exactly. This is why I removed it from the "not reported" section. And thanks for adding it to the issue